main.go

```diff
@@ -271,6 +299,26 @@ func (z *Config) Archive(meeting zoom.Meeting) error {
 	z.logger.Printf("archiving meeting %d to %s (https://drive.google.com/drive/folders/%s)",
 		meeting.ID, meetingFolder.Name, meetingFolder.Id)
 	for _, f := range meeting.RecordingFiles {
+
+		//check if meeting duration is shorter than minimum
+		start, err := time.Parse(time.RFC3339, f.RecordingStart)
```

Comment:
Many times we stop the recording at the beginning of the meeting and start it again later when everyone is ready. This creates short videos that are uploaded because we were only checking the full meeting recording length.